tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/OutlinePossibleValuesService.java

```diff
@@ -15,34 +15,55 @@
 package com.liferay.ide.upgrade.plan.core.internal;
 
 import com.liferay.ide.core.util.SapphireContentAccessor;
+import com.liferay.ide.upgrade.plan.core.IUpgradePlanOutline;
 import com.liferay.ide.upgrade.plan.core.UpgradePlanCorePlugin;
 
-import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 import java.util.Set;
 
-import org.eclipse.core.runtime.Platform;
-import org.eclipse.core.runtime.preferences.IPreferencesService;
 import org.eclipse.sapphire.PossibleValuesService;
 
 /**
  * @author Terry Jia
+ * @author Simon Jiang
  */
 public class OutlinePossibleValuesService extends PossibleValuesService implements SapphireContentAccessor {
 
 	@Override
 	protected void compute(Set<String> values) {
-		IPreferencesService preferencesService = Platform.getPreferencesService();
+		values.addAll(_possibleValues);
+	}
+
+	@Override
+	protected void initPossibleValuesService() {
```

Comment:
We should only show two outline options ("code upgrade" or "database upgrade").  We should show the filename (liferay-docs.zip) as the customer doesn't care about this detail.